Gemfile.lock

```diff
@@ -261,14 +261,14 @@ GEM
       rspec-mocks (~> 3.4.0)
       rspec-support (~> 3.4.0)
     rspec-support (3.4.1)
-    rubocop (0.37.2)
-      parser (>= 2.3.0.4, < 3.0)
+    rubocop (0.40.0)
```

Comment:
We're currently on ruby 2.2.4.  I attempted to go to 2.2.5, but linting fails because of errors with the parser gem.